pkg/codegen/python/gen.go

```diff
@@ -1845,26 +1928,46 @@ func generateModuleContextMap(tool string, pkg *schema.Package, info PackageInfo
 	}
 
 	inputSeen, outputSeen := codegen.Set{}, codegen.Set{}
-	visitObjectTypesFromProperties(pkg.Config, outputSeen, func(t *schema.ObjectType) {
-		getModFromToken(t.Token).details(t).outputType = true
+	visitObjectTypesFromProperties(pkg.Config, outputSeen, func(t interface{}) {
+		switch T := t.(type) {
+		case *schema.ObjectType:
+			getModFromToken(T.Token).details(T).outputType = true
+		case *schema.ResourceType:
+			getModFromToken(T.Token)
```

Comment:
I don't think we need the `case *schema.ResourceType: getModFromToken(T.Token)` in any of the calls to `visitObjectTypes`/`visitObjectTypesFromProperties` inside `generateModuleContextMap`. Did you find there was a need for it?

Reply:
I updated all of the callsites for `visitObjectTypesFromProperties`, but didn't check to see if they were all necessary.

Reply:
Looks like those weren't needed, so removed them.